tldw_Server_API/tests/Claims/test_claim_embeddings_chroma.py

```
@@ -77,3 +80,95 @@ def test_optional_claim_embeddings_with_chroma(monkeypatch):
+        mock_client.get_or_create_collection.return_value = mock_coll
+
+        manager = ChromaDBManager(
+            user_id="test_user",
+            user_embedding_config={
+                "USER_DB_BASE_DIR": base_dir,
+                "embedding_config": {"default_model_id": "unused", "models": {}},
+                "chroma_client_settings": {"backend": "stub"},
+            },
+            client=mock_client,
+        )
+        manager.db_path = "/tmp/claims-ingest-media.db"
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/claims-ingest-media.db"

(S108)